src/webgl/context.js

```diff
@@ -218,21 +219,17 @@ export function withParameters(gl, {scissorTest, framebuffer, nocatch = true}, f
     if (framebuffer) {
       // TODO - was there any previously set frame buffer?
       // TODO - delegate "unbind" to Framebuffer object?
-      gl.bindFramebuffer(gl.FRAMEBUFFER, null);
+      framebuffer.unbind();
     }
   }
 
   let value;
-  if (nocatch) {
```

Comment:
Remove nocatch temporarily as withState() function does not support nocatch path yet